CHANGELOG.md

```diff
@@ -48,8 +48,13 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+* [#13673](https://github.com/cosmos/cosmos-sdk/pull/13673) Fix `--dry-run` flag not working when using tx command.
```

Comment:
@likhita-809, I've missed it but the changelog have been included to the wrong place. Can you put it under unreleased?